slack-lint-checks/src/main/java/slack/lint/denylistedapis/DenyListedApiDetector.kt

```diff
@@ -491,6 +496,14 @@ internal class DenyListedApiDetector : Detector(), SourceCodeScanner, XmlScanner
           ),
       )
     }
+
+    private fun loadExternalEntries(): Set<DenyListedEntry> {
+      val loader = ServiceLoader.load(DenyListedEntryLoader::class.java)
+
+      return mutableSetOf<DenyListedEntry>().apply {
+        loader.iterator().forEachRemaining { addAll(it.entries) }
+      }
```

Comment:
You should be able to do something like `ServiceLoader.load(...).iterator().toSet()`

Reply:
nice, the more concise the better. I'm not seeing an applicable `toSet()` call to use here, but I might be missing something.

Reply:
maybe `.iterator().asSequence().toSet()`